src/connector/src/parser/debezium/simd_json_parser.rs

```diff
@@ -625,4 +658,58 @@ mod tests {
             panic!("the test case is expected to be failed");
         }
     }
+
+    // postgres-specific data-type tests
+    #[tokio::test]
+    async fn test_date_time() {
```

Comment:
The name of this case is too vague, should specify the upstream source

Reply:
I have now grouped this case into the module `test3_postgres` :)